src/shared/locales/en/minisry.json

```diff
@@ -1 +1,67 @@
-
+{
+  "tr_addServiceTime": "Add service time",
+  "tr_hours": "Hours",
+  "tr_bibleStudies": "Individual Bible studies",
+  "tr_btnAdd": "Add",
+  "tr_creditHours": "Credit hours",
+  "tr_addNewStudy": "Add new Bible study",
+  "tr_pioneerSchool": "Pioneer school",
+  "tr_сreditHours": "{{ ministry time }} hours",
```

Comment:
@mrtavor: would it be possible to change this placeholder not to use any space. i.e. `ministryTime`. Thanks